Documentation/building/windows/dependencies.md

```diff
@@ -0,0 +1,44 @@
+# Build Dependencies for Windows
+
+Building Xamarin.Android requires:
+
+  * An existing installation of the Xamarin.Android SDK and the Android SDK
+  * The .NET Framework 3.5 &ndash; 4.7 development tools
+  * Git for Windows
+  * The Java Development Kit (JDK)
+
+The recommended steps to install these dependencies are:
+
+ 1. Run the [Visual Studio Installer](https://visualstudio.microsoft.com/vs/).
+
+ 2. Under the **Workloads** tab, ensure that the **Mobile development with
```

Comment:
Still not sure why this would be *required* to build xamarin-android, given that we *download and install our own Android SDK + NDK* as part of our normal build process.

It may be *desirable* for other reasons to do this, but I don't think it's *required*.

Reply:
One reason we *might* want to keep the mobile development is to provide templates. 
Currently if we are using OSS X.A I guess you have to create the entire project yourself manually. @brendanzagaeski is that correct?